mappings/net/minecraft/client/texture/Sprite.mapping

```diff
@@ -1,2 +1,39 @@
 CLASS net/minecraft/class_4168 net/minecraft/client/texture/Sprite
+	FIELD field_30054 frames Ljava/util/List;
+	FIELD field_30056 rotated Z
+	FIELD field_30057 x I
+	FIELD field_30058 y I
+	FIELD field_30059 width I
+	FIELD field_30060 height I
+	FIELD field_30063 name Ljava/lang/String;
+	FIELD field_30065 u0 F
+	FIELD field_30066 u1 F
+	FIELD field_30067 v0 F
+	FIELD field_30068 v1 F
```

Comment:
uMin, uMax, vMin, vMax

Reply:
these are mojang names

Reply:
mojang names are always correct

Reply:
But we follow upstream, and upstream has those names

Reply:
*fine*